src/middlewared/middlewared/plugins/pool_/dataset_quota_and_perms.py

```diff
@@ -10,181 +10,6 @@ class PoolDatasetService(Service):
     class Config:
         namespace = 'pool.dataset'
 
-    @accepts(
-        Str('id', required=True),
-        Dict(
```

Comment:
FWIW, removing all this code means that now this file has many imported but not used flake8 warnings.
- `errno`
- `Bool`
- `OROperator`
- `UnixPerm`
- `CallError`